<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>683</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Cadastro de clientes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(62, 132, 238);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="InputNome">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-color: rgb(0,0,0);
</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Nome do cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnInserir">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>570</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(78, 154, 6);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Inserir novo cliente</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputCPF">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-color: rgb(0,0,0);
</string>
    </property>
    <property name="inputMask">
     <string>000.000.000-00</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Numero do Celular com DDD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputEmail">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-color: rgb(0,0,0);
</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnGeraCPF">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(114, 159, 207);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Gera CPF</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>InputNome</tabstop>
  <tabstop>InputCPF</tabstop>
  <tabstop>InputEmail</tabstop>
  <tabstop>BtnInserir</tabstop>
  <tabstop>BtnGeraCPF</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
